projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/vdk.py

```diff
@@ -14,8 +16,6 @@ class VdkUI:
     def run_job(path, arguments=None):
         """
         Execute `run job`.
-        When no auth is provided, disables prompts for the password to avoid the terminal hanging.
-        When auth is provided, await prompts for username/passwords and sends them
```

Comment:
nit: what changed functionally, for changing the doc?

Reply:
It is not currently available, I forgot to delete them the last time